app/main/views/drafts.py

```diff
@@ -229,3 +256,58 @@ def publish_draft_service(service_id):
     index_service(new_service)
 
     return jsonify(services=new_service.serialize()), 200
+
+
+@main.route('/draft-services/<string:framework_slug>/create', methods=['POST'])
+def create_new_draft_service(framework_slug):
+    """
+    Create a new draft service with lot, supplier_id, draft_id, framework_id
+    :return: the new draft id and location e.g.
+    HTTP/1.1 201 Created Location: /draft-services/63636
+    """
+    updater_json = validate_and_return_updater_request()
+    draft_json = validate_and_return_draft_request()
+
+    # TODO: Get framework id by matching the slug once in Framework table.
+    framework_id = Framework.query.filter(
+        Framework.name == 'G-Cloud 7'
+    ).first_or_404().id
+
+    # TODO: Reject any requests on a Framework that is not currently 'open'
+
+    supplier_id = draft_json['supplierId']
+    lot = draft_json['lot']
+    errs = get_draft_validation_errors(draft_json, lot, slug=framework_slug)
+    if errs:
+        return jsonify(errors=errs), 400
+
+    draft_json = drop_foreign_fields(draft_json, ['supplierId'])
+    now = datetime.utcnow()
```

Comment:
The best place to do this looks like it would be in the ServiceTableMixin.  The only issue being that there is the chance that the initial updatedAt and createdAt might not be microsecond-equivalent.  Do we care about that?